lib/y2_installbase.pm

```diff
@@ -255,6 +274,50 @@ sub select_all_patterns_by_menu {
     assert_screen 'inst-overview';
 }
 
+=head2 select_not_install_any_pattern 
+
+    select_not_install_any_pattern() 
+
+Being in the "select pattern" screen, performs steps to not install any
+patterns.
+=cut
+
+sub select_not_install_any_pattern {
+    my ($self) = @_;
+
+    # Ensure mouse on certain pattern then right click
+    assert_and_click("minimal-system", button => 'right');
+    assert_screen 'selection-menu';
+    # select action on all patterns
+    wait_screen_change { send_key 'a'; };
+    # confirm do not install
+    assert_and_click 'all-do-not-install';
+    save_screenshot;
+}
+
+=head2 select_current_visible_patterns 
+
+    select_current_visible_patterns()
+
+Being in the "select pattern" screen, performs steps to select visible
+patterns.
+=cut
+
+sub select_current_visible_patterns {
+    my ($self) = @_;
+    my @patterns = split(/,/, get_var('PATTERNS', ''));
+    # remove empty elements
+    @patterns = grep { $_ ne '' } @patterns;
```

Comment:
we should do this outside the function in the test module, I think better with your approach [here](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/15722/files#diff-edbe5e5e59e46c6bbb5beae3b3116a94517b2e181048a157072dab57f72219a8R17)